components/lib/crash/src/main/java/mozilla/components/lib/crash/service/SentryService.kt

```diff
@@ -104,3 +118,11 @@ private fun createMessage(crash: Crash.NativeCodeCrash): String {
 
     return "NativeCodeCrash(fatal=$fatal, minidumpSuccess=$minidumpSuccess)"
 }
+
+private fun createMessage(crash: Crash.UncaughtExceptionCrash): String {
+    return "Fatal crash: ${crash.throwable.message})"
+}
```

Comment:
Won't that mess with our existing crashes since they now start to have a different message? Maybe we should only add a prefix for the new type of "non-crashes"? 🤔

Reply:
Agreed.  Sure, I'll remove the prefix for crashes.  Only do it for non-crashes